### PR TITLE
landscape: new shortcut prefs

### PR DESCRIPTION
**Pull request by patosullivan**
Closes https://github.com/tloncorp/homestead/issues/349

Edit: this should be good to go now outside of the missing keybindings themselves.

Still need:
- [ ] actual shortcuts
- [x] detection of windows/linux vs mac
- [x] design on what the search box should actually do (cc @urcades )

### CODE REVIEW

File: pkg/grid/src/preferences/ShortcutPrefs.tsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        <span className="px-3 py-2 text-gray-400 text-sm font-semibold">Action</span>
+        <span className="px-3 py-2 text-gray-400 text-sm font-semibold">Keybinding</span>
+        {shortcuts
+          .map((shortcut) => ({
+            action: shortcut.action,
+            keybinding: shortcut.keybinding.replace('Ctrl', metaKey)
+          }))
+          .map((shortcut) => ({
+            action: shortcut.action,
+            keybinding: shortcut.keybinding.replace('Alt', altKey)
```
Comment (liam-fitzgerald) [inline pkg/grid/src/preferences/ShortcutPrefs.tsx:93]: any reason this can't be one .map?

File: pkg/grid/src/preferences/ShortcutPrefs.tsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        />
+      </label>
+    </>
+  );
+};
+
+export const ShortcutPrefs = () => {
+  const [searchInput, setSearchInput] = useState('');
+  const [matchingShortcuts, setMatchingShortcuts] = useState<string[]>([]);
+  const metaKey = window.navigator.platform.includes('Mac') ? '⌘' : 'Ctrl';
```
Comment (liam-fitzgerald) [inline pkg/grid/src/preferences/ShortcutPrefs.tsx:72]: does this handle the "iPad w/keyboard" case? they usually have a command key

**Review by liam-fitzgerald — COMMENTED** [review]
some questions, else LGTM!

File: pkg/grid/src/preferences/ShortcutPrefs.tsx (hunk 2 above)
Comment (matildepark) [reply pkg/grid/src/preferences/ShortcutPrefs.tsx:72]: iPads declare themselves macs in the agent.

**Review by urcades — APPROVED** [review]
lgtm

**Pull request merged**